src/Documentation/doc/basic/terminal.md

```diff
@@ -10,7 +10,7 @@ The Terminal contains a very basic filesystem to help you organize files into di
 Note that this is **not** a _true_ filesystem implementation and instead relies almost entirely on string manipulation.
 For this reason, some features found in real-world filesystems do not exist in our Terminal. For example:
 
-- Tab autocompletion does not work with relative paths
+- Tab autocomplete does not work with relative paths
```

Comment:
I think this one was correct as-is.

Reply:
I wasn't sure if autocompletion was a correct word. Of what I could find online, most of these programs are called 'autocomplete'. Please feel free to pick your choice, as these things often are nitpicky and come down to preference.

Reply:
"Autocomplete" is a verb, so a program that does it is an "autocompleter," past-tense is "autocompleted," and the result of doing it is "autocompletion." You won't find much usage online, since it's a niche topic, but it's following the standard rules of English grammar (such as they exist).

Reply:
I understand. I am keeping the original wording.